### DISCUSSION TITLE
connect to MS sql server with integrated using .net framework (c#)

### DISCUSSION THREAD

**Discussion by Abhisheik-athi** [dotnet/runtime#64096] [Q&A]
i am using .net framework and trying to connect to a MS SQL server with integrated security = true.  Need to use different credentials so is there a way using kerberos authentication. and authenticate the creds that is needed for the sql connection ?
need help on how to do it.

**Comment by huoyaoyuan** [comment]
You'd better ask at https://github.com/dotnet/SqlClient . The new SQL server client is developed there.

**Reply by Abhisheik-athi** [reply]
@huoyaoyuan  Is there a way to get NTLM based token in .NET (C#) for a different set of AD username and passowrd ?

**Reply by Abhisheik-athi** [reply]
so that i could pass that token in a web request / establish a jdbc conn.

**Reply by huoyaoyuan** [reply]
I don't know any detail about sql client.
